packages/ubuntu_provision/lib/src/eula/eula_page.dart

```diff
@@ -23,39 +23,27 @@ final eulaPageProvider = FutureProvider<File>((ref) async {
   final localizedEula = File(p.join(directory, 'EULA_$locale.pdf'));
   late final fallbackEula = File(p.join(directory, 'EULA.pdf'));
   final eulaFile = localizedEula.existsSync() ? localizedEula : fallbackEula;
+  assert(eulaFile.existsSync()); // throw error if fallback file doesn't exist
```

Comment:
is this refactor something that is required as part of updating the Flutter version? Not saying we shouldn't do it but it feels like this its own thing.

Reply:
I didn't have the time to debug it thoroughly, but the integration test failed since the `eulaPageProvider` was stuck in a loading state when trying to read a non-existent file. Making the page itself a stateless widget and adding this assertion fixed the behavior, but I didn't find the underlying cause for the test failure (instead of going down a hole of subtle UI state issues, I decided to simplify it a bit)